platform-hub-web/src/app/users/users-list.html

```diff
@@ -72,6 +72,20 @@
           ng-click="$ctrl.revokeAdmin(u)">
           Revoke admin
         </md-button>
+        <md-button
+          ng-if="u.is_active"
+          class="md-primary"
+          aria-label="Deactivate user"
+          ng-click="$ctrl.deactivateUser(u)">
+          Deactivate
+        </md-button>
+        <md-button
+          ng-if="!u.is_active"
+          class="md-primary"
+          aria-label="Activate user"
+          ng-click="$ctrl.activateUser(u)">
+          Activate
+        </md-button>
       </md-card-actions>
     </md-card>
```

Comment:
We should prominently show whether the user is active or not in the listings AND on the user page (assuming we have one). Maybe have a look at how we do badges in the announcements listings and do something similar here?